src/Avatar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { stringToColor } from '../util/colorMapping';
+import { Props } from './props';
+import { styles } from './styles';
+
+export const Avatar = ({
+  avatarUrl = undefined,
+  username = undefined,
+  isOnline = undefined,
+  showOnline = false,
+  style = {},
+  onClick = () => {}
+}: Props) => {
+
+  const text = username ? username.substring(0, 2).toUpperCase() : ''
+  const color = stringToColor(username)
+
+  return (
+    <div style={{ width: '48px', height: '48px' }} onClick={onClick}>
+      <div style={{ height: '0px' }}>
+        <div
+          className='ce-avatar'
```

Comment:
Is there a way to get the same view with using less divs?

I know I wrote this so it's on me, but it'd be great is we can get away with less classNames + divs just to make the UI concise. 

1. Easier for users to learn / reverse-engineer our code
2. Less places people will need to inject their own styles to customize
3. Removes sources of redundancy

Can you try 1 or 2 small experiments for this? If this is the most concise say so and I'll approve

Reply:
yeah this one is big time copy/pasta from me I'll clean it up

Reply:
ooh actually the reason for all the divs is to get the status indicator in the correct spot

Reply:
Okay fair enough